packages/swapper/src/swappers/thorchain/getThorTradeQuote/getTradeQuote.ts

```diff
@@ -59,16 +59,18 @@ export const getThorTradeQuote: GetThorTradeQuote = async ({ deps, input }) => {
       buyAsset.precision,
     )
 
-    const estimatedTradeFee = await estimateTradeFee(deps, buyAsset)
+    const estimatedTradeFeeCrypto = await estimateBuyAssetTradeFeeCrypto(deps, buyAsset)
 
-    const buyAssetUsdRate = await getUsdRate({ deps, input: { assetId: buyAsset.assetId } })
-    const minTradeFeeAmount = MINIMUM_USD_TRADE_AMOUNT.div(buyAssetUsdRate)
```

Comment:
That was incorrect, we were mixing crypto and fiat amounts, we need the minimum trade fee amount to be in USD